UKFast.API.Client.ECloud/Models/Request/CreateVirtualMachineRequest.cs

```diff
@@ -30,50 +31,53 @@ public class CreateVirtualMachineRequest
         [Newtonsoft.Json.JsonProperty("ram")]
         public int RAM { get; set; }
 
-        [Newtonsoft.Json.JsonProperty("hdd")]
-        public int HDD { get; set; }
+        [Newtonsoft.Json.JsonProperty("hdd", NullValueHandling = NullValueHandling.Ignore)]
+        public int? HDD { get; set; }
```

Comment:
Rather than making this nullable, `DefaultValueHandling = DefaultValueHandling.Ignore` can be used instead